scss/_variables.scss

```diff
@@ -684,6 +684,8 @@ $line-height-base:            calc(18 / 16) !default;
 $line-height-sm:              calc(16 / 14) !default;
 $line-height-lg:              calc(30 / 16) !default;
 // stylelint-enable function-disallowed-list
+$small-line-length:           32ch !default;
+$medium-line-length:          65ch !default;
```

Comment:
`ch` unit [seems well supported](https://caniuse.com/ch-unit) but I'm wondering how you chose those values (`32ch` and `65ch`). The design specs talk about 20-40 characters and 60-80 characters.